recipes/django-jsoneditor/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "django-jsoneditor" %}
+{% set version = "0.0.12" %}
+{% set build = 0 %}
+{% set sha256 = "5f38e808cf6148fb38309551e8c76a19c6bff2403aff8456e798fd4944a97127" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/ae/b0/0164e082b29777fbc56c2373b68da93d23a29a040787e7f1c65e1562f133/django-jsoneditor-0.0.12.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: {{ build }}
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - packaging
```

Comment:
Is `packaging` a `build` requirement or a `run` requirement? I see it in the following, which would seem to indicate that you need it at runtime:

https://github.com/nnseva/django-jsoneditor/blob/master/jsoneditor/forms.py#L1

I don't think the import tests would catch this.

Reply:
good catch

```
import: 'jsoneditor.forms'
Traceback (most recent call last):
  File "/home/kwilcox/miniconda3-py36/conda-bld/django-jsoneditor_1515523534627/test_tmp/run_test.py", line 5, in <module>
    import jsoneditor.forms
  File "/home/kwilcox/miniconda3-py36/conda-bld/django-jsoneditor_1515523534627/_t_env/lib/python3.6/site-packages/jsoneditor/forms.py", line 1, in <module>
    from packaging import version
ModuleNotFoundError: No module named 'packaging'
TESTS FAILED: django-jsoneditor-0.0.12-py_0.tar.bz2
```